### ISSUE TITLE
File upload doesn't work for large files

### ISSUE THREAD

**Issue by sap1ens** [theintern/leadfoot#52]
I'm running "#type -> file upload" test from Element.js suite via browserstack. It works when I use default upload.txt file or any file under ~60Kb. If I use any file bigger than ~60Kb I receive the following exception:

```
oversubscribed dynamic bit lengths tree
```

Screenshot:

<img width="717" alt="screen shot 2015-08-28 at 10 49 08 am" src="https://cloud.githubusercontent.com/assets/260702/9553234/7b5641c8-4d72-11e5-9464-80a55072a44b.png">

This exception goes from java.util.zip.ZipException, which is probably returned from Selenium server. It means that zip file is invalid / corrupted (http://stackoverflow.com/questions/3049448/java-util-zip-zipexception-oversubscribed-dynamic-bit-lengths-tree)

I think it can be related to adm-zip library implementation. 

**Steps to reproduce**: 
1) Add any file bigger than 60Kb to /tests/functional/data/ folder
2) Edit "#type -> file upload" test from Element.js suite: replace `upload.txt` with the filename from 1), update expected file size
3) Run "#type -> file upload" test from Element.js suite

**Expected output**: successful test

**Actual output:**

```
[reduced...]nPee767/9+hdQSwECCgAKAAAACAC8VQVHxVtD+n85AAAA5gAAEwAAAAAAAAAAALYBAAAAAAAAMTUwX3RyYW5zYWN0aW9uLnhsc1BLBQYAAAAAAQABAEEAAACwOQAAAAA="}] oversubscribed dynamic bit lengths tree
  at Session._post  </Users/sap1ens/Dev/leadfoot/Session.js:59:30>
  at </Users/sap1ens/Dev/leadfoot/Session.js:1687:16>
  at new Promise  <node_modules/dojo/Promise.ts:411:3>
  at Session._uploadFile  </Users/sap1ens/Dev/leadfoot/Session.js:1681:9>
  at Element.type  </Users/sap1ens/Dev/leadfoot/Element.js:200:24>
  at <tests/functional/Element.js:80:21>
  at <node_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/dojo/Promise.ts:11:11>
  at <node_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/dojo/Promise.ts:237:7>
```

And you can also check the screenshot from Browserstack side.

**Additional details**:
OS X 10.10.4
Leadfoot: master branch

**Comment by sap1ens** [comment]
It seems like adm-zip is broken for some files. JSZip is more reliable, I've created PR to fix it #53

**Comment by sap1ens** [comment]
I agree that 60Kb seems like a weird magic number, but I've spent a few hours testing different file types and sizes, it seems right for my local setup.
